### PR TITLE
Additional context for SIC-001 error.

### PR DESCRIPTION
**Pull request by clstokes**

### CODE REVIEW

File: packer/builds/troubleshooting.html.md [hunk 1]
```diff
@@ -63,7 +63,11 @@ This is a short list of internal errors and what they mean.
 
 - SIC-001: Your data was being ingressed from GitHub but failed
 to properly unpack. This can be caused by bad permissions, improper
-symlinks or very large repository sizes.
+symlinks or very large repository sizes. _**Note:** Most often this error occurs
```
Comment (sethvargo) [inline packer/builds/troubleshooting.html.md]: I would be careful saying "most often", but otherwise LGTM

Comment (pearkes) [inline packer/builds/troubleshooting.html.md]: I'd also add a note about symlinks!

Comment (pearkes) [inline packer/builds/troubleshooting.html.md]: oh lol it's already there

**Comment by pearkes** [comment]
LGTM

**Pull request merged**